native-gradle-plugin/src/main/java/org/graalvm/buildtools/gradle/NativeImagePlugin.java

```diff
@@ -226,7 +232,7 @@ private static NativeImageOptions createMainExtension(Project project) {
         return buildExtension;
     }
 
-    private static NativeImageOptions createTestExtension(Project project, NativeImageOptions mainExtension) {
+    private static NativeImageOptions createTestExtension(Project project, NativeImageOptions mainExtension, File testResultsDir) {
```

Comment:
:x: This shouldn't use a `File` or the configuration is order dependent. I will look into fixing this properly.

Reply:
Thanks